test-case/check-pause-release-suspend-resume.sh

```diff
@@ -75,7 +75,7 @@ OPT_NAME['l']='loop'         OPT_DESC['l']='loop count'
 OPT_HAS_ARG['l']=1             OPT_VAL['l']=5
 
 OPT_NAME['i']='sleep-period' OPT_DESC['i']='sleep period of aplay, unit is ms'
-OPT_HAS_ARG['i']=1             OPT_VAL['i']='100'
+OPT_HAS_ARG['i']=1             OPT_VAL['i']='500'
```

Comment:
can I ask what platforms? This is a 5x increase, but the interval between pause_release and pause_push are rather small.

I also don't get why this is different from the regular pause-resume test.

Reply:
@YongAnLu00 Yongan test the script on his TGLU_SKU0A32_SDCA device and found that 200ms seems still too short, and Marc suggest change the sleep time much longer but still quick like 1se